languages/go/exercises/concept/basic-slices/.docs/instructions.md

```diff
@@ -0,0 +1,42 @@
+# Instructions
+
+As a magician-to-be, Elyse needs to practice some basics. She has a stack of cards that she wants to do some operations on.
+
+To make things a bit easier she uses only the cards 1 to 10.
+
+### 1. Retrieve a card from a stack
+
+Return the card at position `index` from the given stack.
+
+```go
+GetItem([]uint8{1, 2, 4, 1}, 2)
+// Returns: 4
+```
+
+### 2. Exchange a card in the stack
+
+Exchange the card at position `index` with the new card provided and return the adjusted stack.
+Note that this will also change the input slice which is ok.
+
+```go
+SetItem([]uint8{1, 2, 4, 1}, 2, 6)
```

Comment:
I find it hard to see what the individual arguments represent. Maybe something like:

```suggestion
index := 2
new_card := 6
SetItem([]uint8{1, 2, 4, 1}, index, new_card)
```

Or does Go have named arguments that can help?

Reply:
no named arguments, no. Looks more readable, yes :)